http-client/src/main/java/io/micronaut/http/client/netty/ssl/NettyClientSslBuilder.java

```diff
@@ -122,7 +123,9 @@ public final SslContext build(SslConfiguration ssl, HttpVersionSelection version
                 versionSelection.getAlpnSupportedProtocols()
             ));
         }
-
+        if (ssl instanceof ClientSslConfiguration clientSsl && clientSsl.isDisableHostnameVerification()) {
+            sslBuilder.endpointIdentificationAlgorithm(null);
```

Comment:
@ChaimaaeROUAI Does setting the endpoint identification algorithm to `null` disable hostname verification? 
```suggestion
            sslBuilder.endpointIdentificationAlgorithm(null);
```

Reply:
yes